### ISSUE TITLE
TypeError: this.chart.render is not a function

### ISSUE THREAD

**Issue by artayru** [apexcharts/vue-apexcharts#88]
Getting the following errors in browser:

` [Vue warn]: Error in mounted hook: "TypeError: this.chart.render is not a function" found in ---> <Apexcharts> <Anonymous> <Anonymous> <Root>`

`TypeError: this.chart.render is not a function`

I belive I'm not doing anything special. That's code of my component, where I'm using apex:

```
<template>
  <div>
    <!-- some other stuff here -->
    <div>
      <apexcharts
        width="550"
        type="bar"
        :options="chartOptions"
        :series="series"
      ></apexcharts>
    </div>
  </div>
</template>

<script>
  import VueApexCharts from '/node_modules/vue-apexcharts'

  export default {
    components: {
      apexcharts: VueApexCharts
    },
    data() {
      return {
        chartOptions: {
          chart: {
            id: "vuechart-example"
          },
          xaxis: {
            categories: [1991, 1992, 1993, 1994, 1995, 1996, 1997, 1998]
          }
        },
        series: [
          {
            name: "series-1",
            data: [30, 40, 45, 50, 49, 60, 70, 81]
          }
        ]
      }
    }
  }
</script>
```

As you can see, very similar to this: https://codesandbox.io/s/pwwz8009n0

Can you please explain me, what am I doing wrong?

P.S. I also tried to declare apexcharts globally in main.js, like here: https://github.com/apexcharts/vue-apexcharts#usage, but result is the same. 
P.P.S I'm using Parcel.

**Comment by junedchhipa** [comment]
Can you try the latest version of `vue-apexcharts@1.3.4` which I just pushed a couple of minutes ago?
Let me know if it worked or not.

**Comment by artayru** [comment]
Now it's ok, thank you! Sorry for the late reply.

**Comment by Patrick1982** [comment]
> Can you try the latest version of `vue-apexcharts@1.3.4` which I just pushed a couple of minutes ago?
> Let me know if it worked or not.

In version 1.3.4 it works but in the current version the problem occurred again.

**Comment by ZiFFeL1992** [comment]
> > Can you try the latest version of `vue-apexcharts@1.3.4` which I just pushed a couple of minutes ago?
> > Let me know if it worked or not.
> 
> In version 1.3.4 it works but in the current version the problem occurred again.

Did you manage to fix it? We are fine on 1.3.6 but seems to break after upgrading..

**Comment by gabrielxpinheiro** [comment]
I have the latest version and I still have this problem.
